### PR TITLE
Release 2.0.0

### PR DESCRIPTION
**Pull request by chuwy**
- [x] Fix tests. It might have something to do with removed decoder of `SchemaCriterion`

### CODE REVIEW

File: project/Dependencies.scala [hunk 1]
```diff
@@ -19,11 +19,11 @@ object Dependencies {
 
   object V {
     val circe        = "0.13.0"
-    val igluClient   = "1.0.0"
+    val igluClient   = "1.0.1"
     val jodaTime     = "2.10.1"
     val analyticsSdk = "2.0.1"
-    val specs2       = "4.8.0"
-    val scalaCheck   = "1.14.0"
+    val specs2       = "4.9.4"
```
Comment (dilyand) [inline project/Dependencies.scala:25]: Is this related to the iglu client bump? If not, it should probably a separate issue.

Comment (chuwy) [reply project/Dependencies.scala:25]: It kind of is. Or rather maybe it is. I had a cryptic runtime exception in tests (coming from spesc2) after bumping Iglu Client. Tried to bump that lib and it worked afterwards. No idea yet if it was related, but suspect it was.
